lib/shiftzilla/helpers.rb

```diff
@@ -331,6 +331,9 @@ def validated_config_file(raw_cfg)
               if not milestones.has_key?(ms)
                 errors << "Release at index #{list_idx} is missing the '#{ms}' milestone."
               else
+                if milestones[ms].start_with?('today')
+                  next
```

Comment:
You should write a proper test for the format that you're expecting. Minimally, test for the allowed patterns that can follow 'today' in a valid string.

Reply:
Heard.  Regexing it up.